frontend/index.html

```diff
@@ -101,6 +101,11 @@ <h2 id='game-over-header' class="hidden">Game Over!</h2>
               </div>
             </div>
 
+            <div class="row justify-content-center">
+              <div id="typo-alert" class="hidden">
+              </div>
+            </div>
+
```

Comment:
Might not have followed this in the index myself, but anything within a row should be within a col. This is due to the negative padding that rows have, which is balanced by the positive padding within cols.

Reply:
I'll play with this as I look into styling for the alert, good tip